### ISSUE TITLE
Set up jupyterhub/lab/dashboard server

### ISSUE THREAD

**Issue by dopplershift** [Unidata/python-general#8]
Set up a server for playing with jupyterhub, jupyterlab, and the dashboard stuff.

Needs to be dockerized on AWS and Jetstream. Leverage jupyter docker stuff as well as dashboard-specific config.

**Comment by julienchastang** [comment]
Yes, this is at the forefront of my mind as well. Dockerized version would be nice perhaps also containing all Unidata Python projects that have notebooks.

**Comment by dopplershift** [comment]
I'll steal your name (unidata-python-docker), but I'm going to base on the scipy-notebook in the [jupyter/docker-stacks](https://github.com/jupyter/docker-stacks) project; that way we can rely on what they've already done.

**Comment by julienchastang** [comment]
Cool. I did not know about that project. Will it be a top-level Unidata project? Once you have this going, I plan to deploy it on Jetstream (or you can too. I aim to get Unidata techies staff set up on Jetstream very soon).

**Comment by dopplershift** [comment]
Be advised, there's a separate container to run a jupyterhub multi-user server (which uses this container). I'm not sure what to do with that one, since there's various secret keys and whatnot involved.

Thoughts?
